client/src/app/tabs/EngineProfile.js

```diff
@@ -239,6 +235,26 @@ function getExecutionPlatformHash(a) {
   return `${a && a.executionPlatform || 'undefined'}#${a && toSemver(a.executionPlatformVersion) || 'undefined' }`;
 }
 
+function buildStatusBarLabel(engineProfile) {
+
+  const {
+    executionPlatformVersion,
+    executionPlatform
+  } = engineProfile;
+
+  if (!executionPlatformVersion) {
+    return `${ENGINE_LABELS[executionPlatform]}`;
+  } else if (executionPlatformVersion.startsWith('1.')) {
+    return `${ENGINE_LABELS[executionPlatform]} (Zeebe ${toSemverMinor(executionPlatformVersion)})`;
+  } else {
+    return `Camunda Platform ${toSemverMinor(executionPlatformVersion)}`;
+  }
+}
+
+function zeebefyVersion(version) {
+  return version.startsWith('1.') ? 'Zeebe ' + version : version;
+}
```

Comment:
From the name, I would expect that it prefixes `Zeebe` all the time. Maybe `getAnnotatedVersion` fits better. What do you think?

Reply:
Agreed and done